src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaClient.cs

```diff
@@ -16,13 +16,13 @@
 
 namespace Microsoft.Health.SqlServer.Features.Schema.Manager
 {
-    public class SchemaClient : ISchemaClient, IDisposable
+    public class SchemaClient : ISchemaClient
```

Comment:
Since we're removing the `IDisposable` interface, let's remove the methods that provide it's implementation on lines 124 - 136.

Reply:
Yes, this is good advice. Since the code didn't return an error and the tests were working seems to me that is okay. But I forgot to see the disposable implementations.

Thank you.